### DISCUSSION TITLE
Hello, could you add a section in the documentation that includes the key bindings for the Hyprland environment?

### DISCUSSION THREAD

**Discussion by Yakumo-Yukari** [Matt-FTW/dotfiles#35] [General]
After installation, the only way to view the key bindings is by checking the Hyprland configuration file. Is it possible to include them directly in the documentation?

**Comment by Matt-FTW** [comment]
https://github.com/Matt-FTW/dotfiles/blob/main/.github/FAQ.md#are-you-planning-on-documenting-the-keybindings
